resources/core/charts/logging/README.md

```diff
@@ -0,0 +1,49 @@
+
+```
+  _                       _             
+ | |                     (_)            
+ | |     ___   __ _  __ _ _ _ __   __ _ 
+ | |    / _ \ / _` |/ _` | | '_ \ / _` |
+ | |___| (_) | (_| | (_| | | | | | (_| |
+ |______\___/ \__, |\__, |_|_| |_|\__, |
+               __/ | __/ |         __/ |
+              |___/ |___/         |___/ 
+```
+
+## Overview
+This document explains how Kyma installs `OK Log` and `Logspout` in the `kyma-system` namespace, and how to use it to check logs in Kyma.
+
+## Details
+The chart installs the following resources in `kyma-system` namespace:
+* Configmap
+* Daemonset
+* Service
+* Statefulset
+
+**OK Log** is a distributed and coordination-free log management system. It's a light weight system with a simple UI.
+
+**Logspout** is a log router for Docker containers that runs as a daemonset. It attaches to all containers on a node, then routes their logs to **OKLog**. 
+
+
+#### Access logs in OK Log UI
+1. Port-forward port **7650**
+
+```bash
+kubectl -n kyma-system port-forward svc/core-logging-oklog 7650:7650
+```
+2. Access OK Log UI in [here](http://localhost:7650/ui).
+3. Use a plaintext or regex to pull up logs. E.g.  `error`
```

Comment:
Use a plain text or regex editor to pull up logs such as `error`.

Reply:
done